src/components/Footer.js

```diff
@@ -9,14 +9,14 @@ import { Navbar, Nav, NavLink, NavItem } from 'reactstrap';
       <Navbar>
       <Nav>
         <NavItem>
-          <NavLink>@CopyRight Geovanna and Nic</NavLink>
+          <NavLink id="copyright">@CopyRight Geovanna and Nic</NavLink>
         </NavItem>
-        <NavItem>
+        {/* <NavItem>
           <NavLink>New</NavLink>
         </NavItem>
         <NavItem>
           <NavLink>Edit</NavLink>
-        </NavItem>
+        </NavItem> */}
```

Comment:
If you are not gonna use this code go ahead and delete it. If you want to use it recommendation is to just bring it in on the branch associated with that link.